cli/azd/pkg/tools/npm/npm.go

```diff
@@ -14,8 +14,9 @@ import (
 
 type NpmCli interface {
 	tools.ExternalTool
-	Install(ctx context.Context, project string, onlyProduction bool) error
-	Build(ctx context.Context, project string, env []string) error
+	Install(ctx context.Context, project string) error
+	RunScript(ctx context.Context, projectPath string, scriptName string, env []string) error
+	Prune(ctx context.Context, projectPath string, production bool) error
```

Comment:
NIT: I think you ended up not using this?

Reply:
Correct.  We're not using it at the moment.